app/mesh/changelog.md

```diff
@@ -3,6 +3,38 @@ title: Kong Mesh Changelog
 no_version: true
 ---
 
+## 1.9.1
+> Release on **2022/10/10**
+
+Built on top of [Kuma 1.8.1](https://github.com/kumahq/kuma/releases/tag/1.8.1)
+
+### Changes
+
+- Gateway: add support for "retryOn"
+- Added support for evicted Pods
```

Comment:
@lobkovilya What is a "Pod"? I read the related issue in the Kuma release notes, but I wasn't sure what it was.

Reply:
It's a term from Kubernetes https://kubernetes.io/docs/concepts/workloads/pods/, already used in this file but with the lowercase first letter.
```suggestion
- Added support for evicted pods
```